packages/office-ui-fabric-react/src/components/Panel/Panel.base.tsx

```diff
@@ -95,7 +95,7 @@ export class PanelBase extends BaseComponent<IPanelProps, IPanelState> implement
       elementToFocusOnDismiss,
       firstFocusableSelector,
       focusTrapZoneProps,
-      forceFocusInsideTrap,
+      forceFocusInsideTrap = true,
```

Comment:
This prop is tagged as `deprecated` and instead use the `focusTrapZoneProps`. Looks like all the props which were in fact just pass through `FocusTrapZone` props are deprecated. So maybe use this pattern in this case too
https://github.com/OfficeDev/office-ui-fabric-react/blob/c3a82c6667d40550dd0dc67de6e69759f2641438/packages/office-ui-fabric-react/src/components/TeachingBubble/TeachingBubble.base.tsx#L36-L51

Reply:
It's true that Panel's forceFocusInsideTrap prop is tagged as deprecated. But that doesn't mean I should remove it with my change, right?

This change in Panel actually isn't required for my fix. I was just fixing it because I noticed that the documentation said that this should default to true, but it actually wasn't.

Reply:
I didn't mean to remove it. We should leave it as it is without adding a default for it. Instead, if you wanna make a fix to align with the documentation, add the default value into something like `defaultFocusTrapZoneProps` object and then mix it with the `focusTrapZoneProps` prop.

Also, I am worried that because this was never implemented and no one noticed an issue with it. I wonder now what is the actual truth... maybe the fix is to change the documentation saying its default value is false. I am leaving this to the area experts to say if that is an error in the code logic or documentation. @dzearing @jspurlin @dagoff

Reply:
Also, thanks a lot for your contribution and hope to get a quick review from mentioned people.

Reply:
Thanks Vitalius!
hmm, I'm hesitant about changing the documentation to say "forceFocusInsideTrap" defaults to false because the normal use case of a Panel is to force focus inside it.
But yes, I'll definitely leave it to the area experts to make the call here.

Reply:
I know the correct behavior should default to `true`. But, won't it be a breaking change if it starts defaulting to `true`?

Reply:
Taking a quick look it seems as though it's relying on the fact that forceFocusInsideTrap is true by default in FocusTrapZone (so indirectly defaults to true), which it shouldn't do. Since this is deprecated I think updating the documentation to not callout a defaultValue (and you potentially add a note stating that the default aligns with default FocusTrapZone behavior, although that feels borderline too verbose...)

---
In reply to: [241220562](https://github.com/OfficeDev/office-ui-fabric-react/pull/7362#discussion_r241220562) [](ancestors = 241220562)

Reply:
Ok, I looked through this in more detail. It turns out that this should NOT be a breaking change. Before my PR, FocusTrapZone treated an undefined value of forceFocusInsideTrap as "true". My PR keeps this behavior.

I'll make the documentation change.